systemlink/clients/core/_jupyter_http_configuration.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+
+"""Implementation of JupyterHttpConfiguration."""
+
+import os
+
+from systemlink.clients import core
+
+
+class JupyterHttpConfiguration(core.HttpConfiguration):
+    """An :class:`HttpConfiguration` specifically for use in Jupyter notebooks."""
```

Comment:
Should we clarify that we mean Jupyter notebooks running in a SystemLink environment as opposed to a notebook running locally?